.github/workflows/publish_docker_image.yml

```diff
@@ -6,13 +6,33 @@ jobs:
   build_and_publish:
     runs-on: ubuntu-latest
     steps:
-      - name: 'Checkout git repo'
-        uses: actions/checkout@v1
-      - name: Publish Docker Image on Tag
-        uses: docker/build-push-action@v1
+      - name: Checkout git repo
+        uses: actions/checkout@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            genomenexus/gn-spring-boot
+          tags: |
+            type=ref,event=tag
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
it is not clear to me, when reading this yml, why these two items are needed (`Set up QEMU` and `Set up Docker Buildx`). Are these dependencies for `docker/build-push-action@v3`? Maybe add a comment line explaining these two blocks?